packages/paste-core/components/inline-code/src/index.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import type {BoxProps} from '@twilio-paste/box';
+
+export interface InlineCodeProps extends Partial<Omit<HTMLDivElement, 'children'>> {
+  children: string;
+  element?: BoxProps['element'];
+}
+
+const InlineCode = React.forwardRef<HTMLDivElement, InlineCodeProps>(
+  ({children, element = 'INLINE_CODE', ...props}, ref) => {
+    return (
+      <Box
+        as="code"
+        element={element}
+        {...safelySpreadBoxProps(props)}
+        backgroundColor="colorBackground"
+        borderWidth="borderWidth10"
+        borderColor="colorBorderWeaker"
+        borderStyle="solid"
+        borderRadius="borderRadius20"
+        color="inherit"
+        fontFamily="fontFamilyCode"
+        fontSize="fontSize20" //do we want to set this, or should this be 'inherit'?
```

Comment:
thought: this should probably inherit so that this component works alongside larger or smaller font sized text. Same for line-height I think. And we should add some stories to test this component inside `<Text>` components with different sizes set.

Reply:
In addition, what do you all think about passing in fontSize and LineHeight style props as overrides to the default inherits?

Would that be useful or too much?

Reply:
@TheSisb agreed, I'll add a story with Inline Code inside different sized Text.

@SiTaggart hmmm... I mean, since we're make the attributes `inherit`, if someone still wanted the code to be a radically different size from the rest of the text, they could just wrap it in another Text with the size set. To me, that seems like a good enough way to provide control. The simpler this presentational component is, the better imo.

Reply:
Perfect reasoning, excellent!